.github/workflows/test_asset-exchange-fabric.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           fi
         working-directory: samples/fabric/fabric-cli
 
-  asset-exchange-fabric-local:
+  asset-exchange-fabric-semi-local:
```

Comment:
Why do we need this? Which part is non-local here?

Reply:
Semi-local, where we use local interopcc, but with go.mod instead of go.mod.local.
That's the only difference, thus it will catch any issue with interopcc when it will be published.